src/BlogBundle/Posts/2018-03-19-Why-we-changed-MediaConch-license.md

```diff
@@ -0,0 +1,27 @@
+---
+title:  "Why we changed MediaConch's license"
+date:   2018-03-19 18:00:00 CET
+excerpt: "We explain the reasons behind the license change of MediaConch."
+tags: MediaConch, License
+---
+
+# Why we changed MediaConch's license.
+
+During [PREFORMA project](http://www.preforma-project.eu/), we\* had several discussions
+about the licensing of [MediaConch](https://mediaarea.net/MediaConch), which was 100% funded by PREFORMA.
+
+Sometimes it was hard. Some historical context: PREFORMA is a consortium, with a lot of different people from different countries and different backgrounds. Some of these people insisted for having a [copyleft license](https://en.wikipedia.org/wiki/Copyleft) for the project. The license of the project was [GPLv3](https://en.wikipedia.org/wiki/GNU_General_Public_License) and [MPLv2](https://en.wikipedia.org/wiki/Mozilla_Public_License). Initially these licenses were also mandatory for any software library we would use in MediaConch as a dependency. If you know about open source licensing, you can guess the origin of a (not-so-small) problem… it would have meant that nearly no library would have been usable, so we would have had to develop everything, including some features like a database, an XML parser, et cetera, completely "from scratch"… when these features are already covered by well known open source libraries. Lot of open source libraries exist but would not be usable because the license was “wrong". We would have pulled ourselves out of the open source ecosystem -- if we were to say to open source advocates that we can’t use a library because it is BSD/MIT/Apache licensed, we would get strange looks from our peers in open source communities. At this early design and proposal phase during the PREFORMA project, we were 6 organizations in competition with each other, to win the bid to develop the file format validation software. All six competitors agreed that we would quit the project if this unnecessary license restriction was not lifted. The response was that we could use third party open source libraries for “non-core" features, and this crisis was averted (phew).

```

Comment:
Given our mixed audience I worry that the library in "nearly no library would have been usable" could be misinterpreted. Maybe 'software library' at least in the first usage.

Reply:
"All six competitors agreed that we would quit the project if this unnecessary license restriction was not lifted." Was this shared, I remember expressing a shared concern but not certain if all competitors agreed to quitting under those terms.

Reply:
It was implicit if I remember well. OK for rewording it and not doing it explicit.